src/script/components/MessagesList/Message/ContentMessage/MessageQuote.tsx

```diff
@@ -222,34 +216,26 @@ const QuotedMessage: FC<QuotedMessageProps> = ({
 
           {asset.isText() && (
             <>
-              <div
-                role="button"
-                tabIndex={0}
-                className={cx('message-quote__text', {
+              <TextMessageRenderer
+                onClickMsg={handleMsgQuoteClick}
+                text={asset.render(selfId)}
+                msgClass={cx('message-quote__text', {
                   'message-quote__text--full': showFullText,
                   'message-quote__text--large': includesOnlyEmojis(asset.text),
                 })}
-                ref={detectLongQuotes}
-                onClick={event => handleClickOnMessage(asset, event)}
-                onKeyDown={event => handleKeyDown(event, () => handleClickOnMessage(asset, event))}
-                dangerouslySetInnerHTML={{__html: asset.render(selfId)}}
-                dir="auto"
+                isCurrentConversationFocused={focusConversation}
+                asset={asset}
                 data-uie-name="media-text-quote"
+                isQuoteMsg
+                setCanShowMore={setCanShowMore}
+                edited_timestamp={edited_timestamp}
```

Comment:
Try to rename edited_timestamp to current code style

```suggestion
                editedTimestamp={editedTimestamp}
```